### PR TITLE
[RW-5162][risk=no] creating new criteria generation script and adding in ICD9 codes

### PR DESCRIPTION
**Pull request by jscherdin**
- this creates a new criteria generation scripts
- updating ICD9 source codes and adds in the new way of generating counts

### CODE REVIEW

File: api/db-cdr/generate-cdr/make-bq-criteria-tables.sh [hunk 1]
```diff
@@ -0,0 +1,550 @@
+#!/bin/bash
```
Comment (freemabd) [inline api/db-cdr/generate-cdr/make-bq-criteria-tables.sh:1]: Are replacing `generate-cb-criteria-tables.sh` with this file? If so should we delete `generate-cb-criteria-tables.sh`?

Comment (jscherdin) [reply api/db-cdr/generate-cdr/make-bq-criteria-tables.sh:1]: It will eventually replace it. It is a slow re-write. Once this new file is complete, we can drop the old one

File: api/libproject/devstart.rb [hunk 2]
```diff
@@ -1007,7 +1007,7 @@ def generate_cb_criteria_tables(cmd_name, *args)
   op.parse.validate
 
   common = Common.new
-  common.run_inline %W{docker-compose run --rm db-make-bq-tables ./generate-cdr/generate-cb-criteria-tables.sh #{op.opts.bq_project} #{op.opts.bq_dataset} #{op.opts.data_browser} #{op.opts.dry_run}}
+  common.run_inline %W{docker-compose run --rm db-make-bq-tables ./generate-cdr/make-bq-criteria-tables.sh #{op.opts.bq_project} #{op.opts.bq_dataset} #{op.opts.data_browser} #{op.opts.dry_run}}
```
Comment (freemabd) [inline api/libproject/devstart.rb:1010]: Ok, if it's a slow rewrite should we revert this change in case we have to regenerate tables in prod or synth?

Comment (jscherdin) [reply api/libproject/devstart.rb:1010]: yeah. I will make that change
